npm/ng-packs/packages/theme-shared/src/lib/handlers/error.handler.ts

```diff
@@ -43,6 +43,29 @@ export const DEFAULT_ERROR_MESSAGES = {
   },
 };
 
+export const DEFAULT_ERROR_LOCALIZATIONS = {
+  defaultError: {
+    title: 'AbpUi::DefaultErrorMessage',
+    details: 'AbpUi::DefaultErrorMessageDetail',
+  },
+  defaultError401: {
```

Comment:
I don't know if I'm missing something here but as far as I can see that only `defaultError` is used below. How are other messages getting called?

Reply:
My bad, I'll be refactored.